edbdeploy/data/terraform/aws/environments/ec2/add_nickname_to_shell.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+CLUSTERNAME=$1
+NODETYPE=$2
+
+if [ "$#" -lt 3 ]; then
+   echo "You must enter at least 23command line arguments"
+else
+   ARGSCOUNTER=0
+   COUNTER=0
+   for ip in "$@"
+   do
+      if [ "$ARGSCOUNTER" -gt 1 ]; then
+         echo "$ip" "$NODETYPE""$COUNTER"."$CLUSTERNAME".internal | sudo tee -a /etc/hosts
```

Comment:
Do we really want to update /etc/hosts once again? `add_ips_to_hosts.sh` is already in charge of doing this.

Reply:
Adding it via the 'add_ips_to_hosts.sh' causes duplication, so it needs to be accomplished individually